docs/guidelines/content/achievement-set-requirements.md

```diff
@@ -9,17 +9,37 @@ description: This guide outlines the essential requirements for creating and sub
 
 ## Every Achievement Set MUST Have
 
-- All game page information filled out, and game images uploaded (if you're not a full-dev, send the images to the code-reviewers).
-- Game Badges for each achievement (they don't need to be distinct from each other, just don't leave them blank).
-- Content covering up to completion so long as the game can be beaten. Whether it be defeating the final boss, completing a first loop, or completing all puzzles, achievement sets that do not cover at least beating the game are deemed unfinished and therefore subject to demotion. **Arcade-style games where the focus is on high scores (such as Pac-Man and Crystal Castles) are exempt from this rule.**
+- **All game page information filled out** 
+   - Do not just link Publisher, Developer, and Genre hubs: fill out the fields in the dev panel or manage app so that this information shows up on all lists that display it.
```

Comment:
```suggestion
   - Do not just link Publisher, Developer, and Genre hubs: fill out the fields in the dev panel or manage app so that this information appears on all lists that display it.
```